packages/core/docs/guide/cart-and-wishlist.md

```diff
@@ -471,205 +403,161 @@ Cleaning the wishlist can be achieved by `clear` property.
 
 Here is an example of how both composables can be used in real-life applications. We have three components: a product list, a cart, and a wishlist. In the examples, we cover the logic of adding/removing items from the wish list and cart, working with getters, and loading state.
 
-The product list: 
+The product list:
 
 ```vue
 <template>
-    <ul>
-      <li
-        v-for="product in products" :key="product.id"> 
-      >
-        <button
-          @click="addToCart({ product, quantity })"
-        >
-          Add to cart
-        </button>
-        <button
-          @click="addToWishlist({ product })"
-        >
-          Add to wishlist
-        </button>
-        <div
-          :isAddedToCart="isInCart({ product })"
-        >
-        </div>
-        <div
-          :isAddedToWishlist="isInWishlist({ product })"
-        >
-        </div>
-      </li>
-    </ul>
-</template>    
-<script>     
-  import { useCart, useWishlist } from '{INTEGRATION}';
-
-  export default {
-    props: {
-      products: {
-        type: Array,
-        required: true
-      }
-    },
-    setup() {
-      const {
-        addItem: addToCart,
-        isInCart,
-        loading 
-      } = useCart();
-
-      const { 
-        addItem: addToWishlist, 
-        isInWishlist
-      } = useWishlist()
-
-      return {
-        addToCart,
-        isInCart,
-        addToWishlist,
-        isInWishlist,
-        loading
-      };
-    }
-  };
+  <ul>
+    <li v-for="product in products" :key="productGetters.getId(product)">
+      <button @click="addToCart({ product, quantity })">
```

Comment:
The same issue as mentioned above - we loop through products, but only have a single `quantity`

Reply:
In this case, should this quantity be changed? Because it is always 1 item added here I think. We use add to cart button not input with number of items.